### DISCUSSION TITLE
Quasar V1 Change slider color based on HEX

### DISCUSSION THREAD

**Discussion by pratik227** [quasarframework/quasar#14285] [General - Components / Directives / etc]
Is there any way to change the slider color based on HEX ? Out of color pallet basically

https://codepen.io/Pratik__007/pen/MWVMZMq

It's coming from v-model let say.

**Comment by metalsadman** [comment]
you can add your own color class then assign it https://v1.quasar.dev/style/color-palette#adding-your-own-colors https://v1.quasar.dev/quasar-utils/color-utils#create-dynamic-custom-colors.  then using helper you can change it dynamically https://v1.quasar.dev/quasar-utils/color-utils#helper-setbrand
